libsolidity/ast/Types.cpp

```diff
@@ -2308,9 +2310,10 @@ MemberList::MemberMap FunctionType::nativeMembers(ContractDefinition const*) con
 	case Location::Bare:
 	case Location::BareCallCode:
 	case Location::BareDelegateCall:
+	case Location::Transfer:
 	{
 		MemberList::MemberMap members;
-		if (m_location != Location::BareDelegateCall && m_location != Location::DelegateCall)
+		if (m_location != Location::BareDelegateCall && m_location != Location::DelegateCall && m_location != Location::Transfer)
```

Comment:
Hm, so send does not support `.gas()` - perhaps we should not support it for transfer either? That would probably even prevent some unwanted reentrancy problems.